jujugui/static/gui/src/app/app.js

```diff
@@ -490,6 +490,8 @@ YUI.add('juju-gui', function(Y) {
       if (this.get('gisf')) {
         document.body.classList.add('u-is-beta');
       }
+
+      this.defaultPageTitle = 'Juju GUI';
```

Comment:
This should probably be a static const nearer the top of the file, instead of an instance level property. Around ln 37